src/pages/Sign/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { darken, lighten } from 'polished';
+
+const Button = styled.button`
+  margin-bottom: ${props => props.marginBottom || '15px'};
+  width: 100%;
+  height: 44px;
+  font-size: 18px;
+  color: ${props => props.color || 'white'};
+  border: none;
+  ${props => {
+    const selected = props.theme.palette[props.backgroundColor];
```

Comment:
```suggestion
  ${props => {
        const selected = props.theme.palette[props.backgroundColor || 'gray'];
```
OR 연산자 이용해서 defaultProps없이 이렇게 표현할수도 있습니다!
defaultProps는 보통 리액트 컴포넌트에 사용할 것으로 예상되는데 styled-components에 부여되어있어서 다소 어색해보입니다!

```suggestion
  ${({backgroundColor = "gray"})=> {
    const selected = props.theme.palette[backgroundColor];
```
또는 매개변수 자리에서 바로 구조분해할당할 수 있습니다. 구조분해 할당 하면서 기본값 부여할 수 있는 것 이용해서 위처럼 표현할 수도 있습니다.

문법적인 부분이니까 한번 잘 확인해보시고 원하시는 방향으로 적용해보시면 될 것 같습니다!

Reply:
defaultprops 수정했습니다!